diff --git a/scripts/d-and-c/make3rd.x64.bat b/scripts/d-and-c/make3rd.x64.bat
index 62956af..0a24b8c 100644
--- a/scripts/d-and-c/make3rd.x64.bat
+++ b/scripts/d-and-c/make3rd.x64.bat
@@ -5,7 +5,6 @@
 @REM ################################################################################
 @REM started with from : http://wiki.flightgear.org/Howto:Build_3rdParty_library_for_Windows
 @REM ################################################################################
-@REM 20160511 - v1.0.4 - Add PLIB build, and install, through special PLIB-1.8.5.zip with a CMakeLists.txt
 @REM 20160510 - v1.0.3 - Add OpenAL build, and install, through openal-build.bat
 @REM 20160509 - v1.0.2 - Massive updates build3rd.x64.bat, including doing Boost
 @REM Renamed build3rd.x64.bat - v1.0.1 - 20140811
@@ -19,7 +18,8 @@
 @set UZ_OPT=x
 @set MOV_CMD=move
 @set MOV_OPT=
-@set SET_BAT="%ProgramFiles(x86)%\Microsoft Visual Studio 10.0\VC\vcvarsall.bat"
+@set SET_BAT="%ProgramFiles(x86)%\Microsoft Visual Studio 12.0\VC\vcvarsall.bat"
+@set GENERATOR="Visual Studio 12 Win64"
 @set HAD_ERROR=0
 
 @set TMP3RD=3rdParty.x64
@@ -29,7 +29,7 @@
 @REM Uncomment this, and add to config/build line, if you can output to a LOG
 @set BLDLOG= ^>^> %LOGFIL% 2^>^&1
 @set ERRLOG=%WORKSPACE%\error-2.txt
-@set ADD_GDAL=0
+@set ADD_GDAL=1
 @set HAVELOG=1
 
 @if EXIST %TMPDN3RD% (
@@ -40,7 +40,7 @@
 @goto EXIT
 )
 
-@call setupqt64
+@REM call setupqt64
 
 @echo %0: Begin %DATE% %TIME% in %CD% > %LOGFIL%
 @echo # Error log %DATE% %TIME% > %ERRLOG%
@@ -98,7 +98,7 @@ md %WORKSPACE%\%TMP3RD%\lib
 md %WORKSPACE%\%TMP3RD%\include
 )
 
-CALL %SET_BAT% amd64
+CALL %SET_BAT% x86_amd64
 
 @REM TEST JUMP
 @REM GOTO DO_CGAL
@@ -147,11 +147,11 @@ md %TMP_BLD%
 
 CD %TMP_BLD%
 
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build" %BLDLOG%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build" %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build"
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build"
 )
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build" %BLDLOG%
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\zlib-build\build" %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit config/gen %TMP_SRC%
@@ -270,12 +270,12 @@ MD %TMP_BLD%
 )
 
 CD %TMP_BLD%
-ECHO Doing 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
+ECHO Doing 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
+ECHO Doing 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
 )
 
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libpng-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake config/gen %TMP_SRC%
@@ -369,7 +369,7 @@ REN %TMP_DIR% %TMP_SRC%
 CD %TMP_SRC%
 
 @IF NOT EXIST jconfig.h (
-@echo Doing 'nmake -f makefile.vc setup-v10'
+@echo Doing 'nmake -f makefile.vc setup-v12'
 nmake -f makefile.vc setup-v10
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
@@ -446,11 +446,11 @@ MD %TMP_BLD%
 )
 
 CD %TMP_BLD%
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
 )
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%" %BLDLOG%
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcurl-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%" %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake conf/gen %TMP_SRC%
@@ -510,11 +510,11 @@ if NOT EXIST %TMP_SRC%\nul (
 )
 
 CD %TMP_SRC%
-ECHO Doing: 'nmake -f makefile.vc MSVC_VER=1600 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES' %BLDLOG%
+ECHO Doing: 'nmake -f makefile.vc MSVC_VER=1800 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing: 'nmake -f makefile.vc MSVC_VER=1600 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES' to %LOGFIL%
+ECHO Doing: 'nmake -f makefile.vc MSVC_VER=1800 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES' to %LOGFIL%
 )
-nmake -f makefile.vc MSVC_VER=1600 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES %BLDLOG%
+nmake -f makefile.vc MSVC_VER=1800 GDAL_HOME=%WORKSPACE%/libgdal-source BINDIR=%WORKSPACE%\%TMP3RD%\bin LIBDIR=%WORKSPACE%\%TMP3RD%\lib INCDIR=%WORKSPACE%\%TMP3RD%\include WIN64=YES %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit nmake building source %TMP_SRC% in %CD%
@@ -532,6 +532,7 @@ xcopy %WORKSPACE%\libgdal-source\alg\gdal_alg.h %WORKSPACE%\%TMP3RD%\include\ /y
 xcopy %WORKSPACE%\libgdal-source\alg\gdalwarper.h %WORKSPACE%\%TMP3RD%\include\ /y /f
 xcopy %WORKSPACE%\libgdal-source\frmts\vrt\gdal_vrt.h %WORKSPACE%\%TMP3RD%\include\ /y /f
 xcopy %WORKSPACE%\libgdal-source\ogr\ogr*.h %WORKSPACE%\%TMP3RD%\include\ /y /f
+xcopy %WORKSPACE%\libgdal-source\ogr\ogrsf_frmts\ogrsf_frmts.h %WORKSPACE%\%TMP3RD%\include\ /y /f
 xcopy %WORKSPACE%\libgdal-source\port\cpl*.h %WORKSPACE%\%TMP3RD%\include\ /y /f
 xcopy %WORKSPACE%\libgdal-source\gdal_i.lib %WORKSPACE%\%TMP3RD%\lib\ /y /f
 xcopy %WORKSPACE%\libgdal-source\gdal.lib %WORKSPACE%\%TMP3RD%\lib\ /y /f
@@ -590,11 +591,11 @@ md %TMP_BLD%
 )
 
 cd %TMP_BLD%
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%"' to %LOGFIL%
 )
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%" %BLDLOG%
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libfltk-build\build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%\%TMP3RD%" %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake conf/gen %TMP_SRC%
@@ -635,7 +636,7 @@ IF %HAVELOG% EQU 1 (
 @REM ECHO However this is ONLY obtaining the simpits boost and binaries %BLDLOG%
 @REM ECHO while download_and_compile obtains the SVN source and does a compile %BLDLOG%
 @REM ECHO So this is presently SKIPPED %BLDLOG%
-@REM GOTO DN_BOOST
+@GOTO DN_BOOST
 @REM :DO_BOOST2
 
 @REM set TMP_URL=http://flightgear.simpits.org:8080/job/Boost-Win64/lastSuccessfulBuild/artifact/*zip*/archive.zip
@@ -717,7 +718,7 @@ IF %HAVELOG% EQU 1 (
 @set TMP_SRC=libcgal-source
 @set TMP_BLD=libcgal-build
 @set TMP_DIR=CGAL-4.8
-@set TMP_PRE=%WORKSPACE%\cgal-source\auxiliary\gmp;%WORKSPACE%\Boost;%WORKSPACE%\install\Boost;%WORKSPACE%\%TMP3RD%
+@set TMP_PRE=%WORKSPACE%\cgal-source\auxiliary\gmp;%BOOST_DIR%;%WORKSPACE%\%TMP3RD%
 
 @if NOT EXIST %TMP_ZIP% (
 @echo Moment, doing 'CALL %GET_EXE% %TMP_URL% %GET_OPT% %TMP_ZIP%'
@@ -779,7 +780,7 @@ CD %TMP_BLD%
 )
 
 @REM -DZLIB_LIBRARY="%WORKSPACE%\%TMP3RD%\lib\zlib.lib" -DZLIB_INCLUDE_DIR="%WORKSPACE%\%TMP3RD%\include" 
-@set TMP_OPS=-G "Visual Studio 10 Win64" -DCMAKE_PREFIX_PATH="%TMP_PRE%" -DCGAL_Boost_USE_STATIC_LIBS:BOOL=ON -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcgal-build\build"
+@set TMP_OPS=-G %GENERATOR% -DCMAKE_PREFIX_PATH="%TMP_PRE%" -DCGAL_Boost_USE_STATIC_LIBS:BOOL=ON -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libcgal-build\build"
 
 @ECHO Doing: 'cmake ..\%TMP_SRC% %TMP_OPS% %BLDLOG%
 IF %HAVELOG% EQU 1 (
@@ -877,11 +878,11 @@ MD %TMP_BLD%
 )
 
 CD %TMP_BLD%
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"' %BLDLOG%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"' to %LOGFIL%
+ECHO Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"' to %LOGFIL%
 ) 
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%/freetype-build/build" -DCMAKE_PREFIX_PATH:PATH="%WORKSPACE%/%TMP3RD%"
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake conf/gen %TMP_SRC%
@@ -1016,11 +1017,11 @@ md %TMP_BLD%
 )
 
 cd %TMP_BLD%
-@echo Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build"' %BLDLOG%
+@echo Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build" -DNOMINMAX:BOOL=ON' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-@echo Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build"' to %LOGFIL%
+@echo Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build" -DNOMINMAX:BOOL=ON' to %LOGFIL%
 )
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build" %BLDLOG%
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libgeos-build\build" -DNOMINMAX:BOOL=ON %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake conf/gen %TMP_SRC%
@@ -1098,11 +1099,11 @@ md %TMP_BLD%
 )
 
 cd %TMP_BLD%
-@echo Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build"' %BLDLOG%
+@echo Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build"' %BLDLOG%
 IF %HAVELOG% EQU 1 (
-@echo Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build"' to %LOGFIL%
+@echo Doing: 'cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build"' to %LOGFIL%
 )
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build" %BLDLOG%
+cmake ..\%TMP_SRC% -G %GENERATOR% -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\libexpat-build\build" %BLDLOG%
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
 @echo %HAD_ERROR%: Error exit cmake conf/gen %TMP_SRC%
@@ -1127,94 +1128,13 @@ xcopy %WORKSPACE%\libexpat-build\build\include\* %WORKSPACE%\%TMP3RD%\include /s
 :DN_EXPAT
 cd %WORKSPACE%
 
-:DO_PLIB
-@set _TMP_LIBS=%_TMP_LIBS% PLIB
-@call :SET_BOOST
-@echo %0: ############################# Download ^& compile PLIB %BLDLOG%
-IF %HAVELOG% EQU 1 (
-@echo %0: ############################# Download ^& compile PLIB to %LOGFIL%
-)
-
-@set TMP_DIR=PLIB-1.8.5
-@set TMP_ZIP=%TMP_DIR%.zip
-@set TMP_URL=http://geoffair.org/tmp/%TMP_ZIP%
-@set TMP_SRC=plib-source
-@set TMP_BLD=plib-build
-
-@if NOT EXIST %TMP_ZIP% ( 
-CALL %GET_EXE% %TMP_URL% %GET_OPT% %TMP_ZIP%
-)
-
-@if NOT EXIST %TMP_SRC%\nul (
-@if NOT EXIST %TMP_DIR%\nul (
-CALL %UZ_EXE% %UZ_OPT% %TMP_ZIP%
-)
-@REM Seems NEED a delay after the UNZIP, else get access denied on the renaming???
-CALL :SLEEP1
-REN %TMP_DIR% %TMP_SRC%
-)
-
-@if NOT EXIST %TMP_SRC%\nul (
-@set /A HAD_ERROR+=1
-@echo %HAD_ERROR%: Failed to set up %TMP_SRC%
-@echo %HAD_ERROR%: Failed to set up %TMP_SRC% >> %ERRLOG%
-@goto DN_PLIB
-)
-
-cd %WORKSPACE%
-
-@if NOT EXIST %TMP_BLD%\nul (
-md %TMP_BLD%
-)
-
-CD %TMP_BLD%
-
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\plib-build\build" %BLDLOG%
-IF %HAVELOG% EQU 1 (
-ECHO Doing: 'cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\plib-build\build"
-)
-cmake ..\%TMP_SRC% -G "Visual Studio 10 Win64" -DCMAKE_INSTALL_PREFIX:PATH="%WORKSPACE%\plib-build\build" %BLDLOG%
-@if ERRORLEVEL 1 (
-@set /A HAD_ERROR+=1
-@echo %HAD_ERROR%: Error exit config/gen %TMP_SRC%
-@echo %HAD_ERROR%: Error exit config/gen %TMP_SRC% >> %ERRLOG%
-)
-
-ECHO Doing 'cmake --build . --config Debug --target INSTALL' %BLDLOG%
-IF %HAVELOG% EQU 1 (
-ECHO Doing 'cmake --build . --config Debug --target INSTALL'
-)
-cmake --build . --config Debug --target INSTALL %BLDLOG%
-@if ERRORLEVEL 1 (
-@set /A HAD_ERROR+=1
-@echo %HAD_ERROR%: Error exit debug building source %TMP_SRC%
-@echo %HAD_ERROR%: Error exit debug building source %TMP_SRC% >> %ERRLOG%
-)
-
-ECHO Doing 'cmake --build . --config Release --target INSTALL' %BLDLOG%
-IF %HAVELOG% EQU 1 (
-ECHO Doing 'cmake --build . --config Release --target INSTALL'
-)
-cmake --build . --config Release --target INSTALL %BLDLOG%
+@REM external builds
+@if EXIST openal-build.bat (
+call openal-build.bat
 @if ERRORLEVEL 1 (
 @set /A HAD_ERROR+=1
-@echo %HAD_ERROR%: Error exit building source %TMP_SRC%
-@echo %HAD_ERROR%: Error exit building source %TMP_SRC% >> %ERRLOG%
+@set _TMP_BLD_FAIL=%_TMP_BLD_FAIL% OpenAL
 )
- 
-xcopy %WORKSPACE%\plib-build\build\include\* %WORKSPACE%\%TMP3RD%\include /y /s /q
-xcopy %WORKSPACE%\plib-build\build\lib\*.lib %WORKSPACE%\%TMP3RD%\lib /y /q
-@REM xcopy %WORKSPACE%\plib-build\build\bin\zlib.dll %WORKSPACE%\%TMP3RD%\bin /y /q
-
-:DN_PLIB
-
-@REM external builds
-@REM if EXIST openal-build.bat (
-@REM call openal-build.bat
-@REM @if ERRORLEVEL 1 (
-@REM @set /A HAD_ERROR+=1
-@REM @set _TMP_BLD_FAIL=%_TMP_BLD_FAIL% OpenAL
-@REM )
 
 :END
 cd %WORKSPACE%
@@ -1300,7 +1220,7 @@ IF %HAVELOG% EQU 1 (
 @goto :EOF
 
 :SET_BOOST
-@set Boost_DIR=%WORKSPACE%\Boost
+@set Boost_DIR=%BOOST_ROOT%
 @echo Set ENV Boost_DIR=%Boost_DIR% %BLDLOG%
 @REM could also use BOOST_ROOT and BOOSTROOT to find Boost.
 @goto :EOF
